modules/machine-user-provisioned-rhv.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * machine_management/user_infra/adding-rhv-compute-user-infra.adoc
+
+:_content-type: PROCEDURE
+[id="machine-user-provisioned-rhv_{context}"]
+= Adding more compute machines to a cluster on {rh-virtualization}
+
+.Procedure
+
+. Modify the `inventory.yml` file to include the new workers.
+. Run the `create-templates-and-vms` Ansible playbook to create the disks and virtual machines.
```

Comment:
Since commands follow this statement, shouldn't it be concluded with a colon?

. Run the `create-templates-and-vms` Ansible playbook to create the disks and virtual machines: